HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+  \begin{tabular}{@{}ll@{}}
+
+  \label{milestones-table}
+    \textbf{Task} & \textbf{Complete Date} \\
+    \midrule
+    Review and Write Up of Current Systems & September 2nd Week \\
+    Mechanical Constraints for Development & September 2nd Week \\ 
+    Sensor Acquisition, Readout Config & October 2nd Week \\
+    Power System and Communications Design & October 2nd Week \\
+    Mechanical Design \footnote{Mechanical deadlines assume rebuilding the box to the HAB4 specification. This work is truncated if the HAB4 structure is not damaged from previous flights.} & October 2nd Week \\
+    Camera Setup and Software & October 2nd Week \\
+    Power System Assembly & October 4th Week \\
+    Mechanical Construction & October 4th Week \\
+    Image Processing Implementation, Testing & December \\
+    APRS Testing and Verification\footnote{Assumes the HAB4 power board is supplying power and APRS to HAB5 avionics boards.} & December \\
+    Testing of All Subsystems & February 2nd Week \\
+    Image Processing Tuning & February 2nd Week \\
+    Assembly and Testing, incl. Environmental & February 4th Week \\
+    Launch & Imagine RIT \\
+  \end{tabular}
+\end{minipage}
+\end{table}
+
+\section{Externalities}
+\subsection{Prerequisite Skills}
+Proficiency in Python (CS 1 or equivalent) is required for implementing WUAP2 image capturing and processing software. 
+Experience with Python computer vision and machine learning tools is required for developing the advanced CV objectives like object detection and segmentation. 
+EE skill set varies greatly based on method of implementation---purchasing ready-to-assemble products to design and fabrication of custom PCBs. 
+The result is that we have the opportunity to tune implementation based on the EEs we find. 
+Understanding of basic circuits and connections/interfacing is still required. 
+An understanding of the fundamentals of heat transfer is recommended in order to do basic thermal analysis. 
+The HAB structure leverages HAB4 designs so the brunt of mechanical design will focus on camera mounting and enclosures. 
+University Physics II (with optics) is recommended for imaging systems testing and calibration tasks.
+
+\subsection{Funding Requirements}
+% Payload:  ~$200 (hard cap at $300)
+% 2x raspberry pi 3 = $100 @ $40-50 each
+% 1x pi camera v2 = $28
+% 1x pi camera noir = $25
+% An arduino FPGA is a possible but unlikely inclusion, with a price tag of $60. The present budget also leaves room for the physical hardware of the payload including camera mounts and possible printing expenses. A filter setup, a la TetraPi, is deemed to be prohibitively expensive.
+
+% 	1x vis filter (reddish) $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB650-40 
+% 	1x nir filter = $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB880-70 
+
+This estimate is based off of the inclusion of a prior HAB board for baseline functionality, an existing APRS module and some functional WUAP-I hardware. 
```

Comment:
There is no listed budget amount.

Also what was the actual cost of HAB4? How much more or less would HAB5 be? What would the cost be if no components are reused? What would it cost if significant elements of HAB4 can be reused?

Reply:
Helium = $100
Balloon = $120
Cameras = $60
RaspPI = $100
Avionics = $150

TOTAL = $530